designs/administrator/future-openshift/resource-related-quick-starts/index.md

```diff
@@ -0,0 +1,65 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Surfacing resource-related quick starts
+
+## Introduction
+Sometimes quick starts can be closely associated with a resource type, for example, a virtual machine or a pipeline. With that in mind, it became important to surface quick starts on that resource's page to improve the flow of both creating a resource and using a resource. This design demonstrates how to do so.
+
+## Design summary
+The design is broken up into scenarios:
+- [Populated resource list views](#populated-resource-list-views)
+- [Empty resource list views](#empty-resource-list-views)
+
+Whether the list view is populated or not, the general pattern of this design includes surfacing a dismissible label to make it clear that a quick start is available within the page heading.
+![general-design](img/general-design.png)
+
+The label is more like a hint and can be hidden, which would then hide **all** resource-related quick start labels for that user.
+![general-design-hide](img/general-design-hide.png)
+
+If there are multiple quick starts associated with a resource, they should be shown in a side panel, where the order logic should be as follows:
+- In progress quick starts
+- Not started quick starts
+- Do not show completed quick starts
+![side-panel](img/side-panel.png)
+
+At the end of the list, there should be a link to view all quick starts associated with that resource. The button should take the user to the quick starts catalog page that is filtered by the resource type.
+![side-panel-cta](img/side-panel-cta.png)
+
+## Empty resource list views
+If a quick start exists that will help the user create a resource type, it should be made available within the empty state of the list view.
+![empty-list](img/empty-list.png)
+
+Selecting the button will immediately start the quick start on the page.
+![empty-list-click-single](img/empty-list-click-single.png)
+
+If multiple quick starts exist that will help the user create a resource type, a side panel should list the quick starts.
+![empty-list-click-multiple](img/empty-list-click-multiple.png)
+
+Choosing to resume or start a quick start from the side panel will resume or start the quick start on that page.
+
```

Comment:
I like how when there are multiple quick starts we are showing a "Getting started with..." button. Do we know what this button label will be tied to? It seems like it needs to be tied to the quick start type = virtual machines (or something like that). I think this one-to-many relationship will need to be designed.

Also, I think I like this "Getting started with..." button label for all quick starts that show up in the empty state like this. It is more clear than the previous example where there is one quick start and we just display the name of the quick start in the button.

Reply:
@beaumorley We just opened a CONSOLE story for this and they were talking about adding labels to the CRD to be able to connect the quick start to one or more resources